runelite-client/src/main/java/net/runelite/client/ui/PluginToolbar.java

```diff
@@ -104,6 +104,10 @@ private void onClick(NavigationButton button)
 			current.setSelected(true);
 
 			PluginPanel pluginPanel = panelSupplier.get();
+			if (!ui.isSideBarShown())
```

Comment:
You can check for this boolean directly in the expandSideBar method and just call the method directly

Reply:
Maybe adding some boolean also to method param would do it